manifests/params.pp

```diff
@@ -304,4 +297,10 @@
   $postgresql_conf_path = pick($postgresql_conf_path, "${confdir}/postgresql.conf")
   $recovery_conf_path   = pick($recovery_conf_path, "${datadir}/recovery.conf")
   $default_database     = pick($default_database, 'postgres')
+
+  $service_reload = if $facts['service_provider'] == 'systemd' {
+    "systemctl reload ${service_name}"
+  } else {
+    "rcctl ${service_name}"
```

Comment:
Does `rcctl` reload? The name implies it's similar to `systemctl` in that it controls `rc`.

Reply:
I was still on it, there was a `reload` missing